typescript/sdk/src/consts/chains.ts

```diff
@@ -24,9 +24,9 @@ export enum Chains {
   mantapacific = 'mantapacific',
   moonbasealpha = 'moonbasealpha',
   moonbeam = 'moonbeam',
+  neutron = 'neutron',
```

Comment:
These should be alphabetical, why move it up?

Reply:
mistake